uber/site_sections/preregistration.py

```diff
@@ -487,12 +487,6 @@ def confirm(self, session, message='', return_to='confirm', undoing_extra='', **
             'affiliates':    session.affiliates()
         }
 
-    def guest_food(self, session, id):
```

Comment:
I couldn't find any references to this URL except in robots.txt, so I removed it.